pydoctor/model.py

```diff
@@ -518,7 +518,7 @@ def _warning(self, current, message, detail):
         if current is not None:
             fn = current.fullName()
         else:
-            fn = '<None>'
+            fn = '.'
```

Comment:
I went with a '.' instead of <None>

This is expected to be the full path to a file for which the report is reported...but we don't have any file for top level project warnings.

Reply:
Do we still need this change? I didn't like how the "guessed project name" message contained `<None>`, but that message is no longer a warning.

I much prefer actual file paths + line numbers where possible and have been slowly making more and more code to log those instead of `fullName`. So I think that in the long run, the `_warning()` method is going to disappear.

Reply:
I will revert this. Thanks!